lib/emerald/grammar/emerald.tt

```diff
@@ -63,25 +63,19 @@ grammar Emerald
     ( variable / !"\n" . )* <InlineLiteral>
   end
 
-  # tag space* string (0 or 1), attr_list (0 or 1)
-  # fix this.
   rule tag_statement
-    tag lparen attributes rparen <TagStatement> /
-    tag space? (( !"\n" . )*)? space* attr_list? <TagStatement>
+    tag space? (( !lparen !newline . )*)? attr_list? <TagStatement>
   end
 
-  # attribute list for
-  # lparen newline (or none), attribute* rparen
   rule attr_list
-    lparen newline attribute* space* rparen <AttributeList>
+    lparen newline lbrace newline attributes rbrace newline rparen <AttributeList>
   end
 
   rule attributes
     attribute* <Attributes>
   end
 
-  # if newline option, must indent
   rule attribute
-    space* attr space* equals space* string comma? <Attribute>
+    attr space* string newline
```

Comment:
I feel like it's good to have consistency, so if you have the choice of doing multiline text in one area, you'd also have the choice of doing it that way elsewhere. thoughts?